tiny_dnn/core/framework/tensor.h

```diff
@@ -172,6 +172,42 @@ class Tensor {
   }
 
   /**
+   * Access to indexes in tensor with or mixed vector/indices
+   * @param vec first indices
+   * @param args rest of indices
+   * @return the value of a specified index in the tensor
+   */
+  template <typename... Args>
+  U host_at(std::vector<size_t> vec, const Args... args) const {
```

Comment:
but the vector is changed, that's kinda counter-intuitive if we'll actually change it